retis-events/src/common.rs

```diff
@@ -27,6 +29,8 @@ impl EventFmt for StartupEvent {
 
         write!(f, "Retis version {}", self.retis_version)?;
 
+        write!(f, "{sep}Command line {}", self.command_line)?;
```

Comment:
This might be a long line, especially if we use the result command from `clap` after profile processing (see the other comment). Is that an issue? Should we have a verbose flag?